build/NuGetPackages/Microsoft.Build.Framework.nuspec

```diff
@@ -26,7 +30,5 @@
     <file src="$configuration$\Output\$id$.xml" target="lib\net46\$id$.xml" />
     <file src="$configuration$-NetCore\Output\$id$.dll" target="lib\netstandard1.3\$id$.dll" />
     <file src="$configuration$-NetCore\Output\$id$.xml" target="lib\netstandard1.3\$id$.xml" />
-    <file src="$configuration$\Output\ref\$id$.dll" target="ref\net46\$id$.dll" />
-    <file src="$configuration$-NetCore\Output\ref\$id$.dll" target="ref\netstandard1.3\$id$.dll" />
```

Comment:
The lib\netstandard1.3 will remain which should give consumer's the same functionality.  We added ref's originally because our libs weren't actually .NETStandard1.3 compatible.  But with this change, they are.